pages/index.html

```diff
@@ -43,48 +44,44 @@ <h1 class="display-5">Add Items</h1>
             <form action="#" class="form-inline">
               <input id= "add-txt" type="text" class="form-control" placeholder="Enter title" autocomplete="off">
               <button id="add-btn" class="btn btn-outline-success ml-2" type="submit">Add</button>
+             
+            
+             </div>
+              
+               
+            </div>
+
+
+
+
             </form>
           </div>
+
+
+    <div class="container" id="dupli">
+      
+    </div>
+    <div class="container" id="success">
+      
+    </div>
+  
           <div class="col-sm-6">
             Lorem, ipsum dolor sit amet consectetur adipisicing elit. Molestias officiis atque eveniet dignissimos, nisi, corrupti nam dolorem autem fugit porro nesciunt eaque, cum numquam temporibus. Provident optio nemo quas consequuntur.
           </div>
         </div>
       </div>
     </div>
-    <div class="container" >
+    <div class="container">
       <div id="titles" class="card-list">
         <div id="title-message" class="card">
           <div class="card-body text-center">
             Your titles will appear here
           </div>
-
         </div>
       </div>
     </div>
-    <div style="color:white">as</div>
-    <div style="color:white">as</div>
     <!-- Scripts -->
-    <!-- jQuery Script -->
-    <script type="../scripts/bootstrap-js/jquery-3.4.1.js"></script>
-
     <script src="../scripts/index.js"></script>
-    <!-- Bootstrap Script -->
-
-    <script src= "../scripts/bootstrap-js/bootstrap.min.js"></script>
-    
-    <!-- Footer -->
-    <footer>
-      <div style="text-align:center">
-
-        <span style="float:left;">
-        <b>   LiBook</b> <i>made by</i> <b >GeekHaven.  </b>
-        </span>
-        <span style="float:right;">
-        <b>Contact:</b> +91 9730000080 , 9000087608
-        </span>
-        <i>© OpenCode Participants</i>
-      </div>
-    </footer>
```

Comment:
This footer along with some other function will get removed if I merge your pr @ananya-mohanty. Can you please check why?

Reply:
Altered that. Please check now!